keps/sig-node/4438-container-restart-termination/README.md

```diff
@@ -207,6 +207,15 @@ the restart during Pod termination.
 List the specific goals of the KEP. What is it trying to achieve? How will we
 know that this has succeeded?
 -->
+The following behaviors should be maintained during pod termination:
+- sidecar containers restarting
+- startupProbe and readinessProbe probing
```

Comment:
How about maintaining the liveness probe as well?

Reply:
Actually,
- all probes should keep running for sidecar containers
- readiness probe should keep running for regular containers (which is the current behavior so we don't have to fix anything about the regular containers)

Reply:
let's get @SergeyKanzhelev agreement in a few hours

Reply:
termination for sidecars should be like normal lifecycle, so all probes should be maintained